_build/src/xml/GUM_news_warhol.xml

```diff
@@ -75,7 +75,7 @@ Warhol	NP	Warhol
 <figure>
 <caption>
 <s type="frag">
-Opening	VVG	open
+Opening	NN	open
```

Comment:
Opening night etc. are VVG (night which opens the show); "The Thursday opening" etc. is NN (note the article)